test_runner/src/main/kotlin/ftl/run/platform/android/GetInstrumentationShardChunks.kt

```diff
@@ -0,0 +1,54 @@
+package ftl.run.platform.android
+
+import com.linkedin.dex.parser.DexParser
+import ftl.args.AndroidArgs
+import ftl.args.ArgsHelper
+import ftl.args.ShardChunks
+import ftl.config.FtlConstants
+import ftl.filter.TestFilter
+import ftl.filter.TestFilters
+import ftl.run.model.InstrumentationTestApk
+import ftl.util.FlankTestMethod
+import ftl.util.downloadIfNeeded
+import java.io.File
+
+fun getInstrumentationShardChunks(
+    args: AndroidArgs,
+    testApks: List<InstrumentationTestApk>
+): Map<InstrumentationTestApk, ShardChunks> =
+    getFlankTestMethods(
+        testApks = testApks.download(),
```

Comment:
It would be great to have concurrent download but I guess it's for another PR :)
cc @bootstraponline

Reply:
I was thinking about concurrent download, but decided to leave as is due to additional changes in tests. But I can add it easily. @bootstraponline ?

Reply:
I think concurrent downloads would be nice